webdriver/commands/Accept_Alert.json

```diff
@@ -0,0 +1,62 @@
+{
+  "webdriver": {
+    "commands": {
+      "Accept_Alert": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/WebDriver/Commands/Accept_Alert",
+          "support": {
+            "webview_android": {
+              "version_added": false,
+              "notes": "To enable Chromium support, include <code>{\"goog:chromeOptions\": {\"w3c\": true}}</code> in the capabiliites when running the command."
+            },
+            "chrome": {
+              "version_added": false,
+              "notes": "To enable Chromium support, include <code>{\"goog:chromeOptions\": {\"w3c\": true}}</code> in the capabiliites when running the command."
+            },
+            "chrome_android": {
+              "version_added": false,
+              "notes": "To enable Chromium support, include <code>{\"goog:chromeOptions\": {\"w3c\": true}}</code> in the capabiliites when running the command."
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "55"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": "6"
```

Comment:
This bit seems dubious. @andreastt, can you confirm IE never implemented WebDriver?

Reply:
There exists a community-supported spec-conforming implementation for IE >= 7. According to https://github.com/SeleniumHQ/selenium/wiki/InternetExplorerDriver it seems it does no longer support IE 6 as I told @chrisdavidmills.

By contrast, the vendor-supported implementation for Edge is not spec-conforming so we cannot reasonably claim that it is compatible.

Reply:
Thanks @andreastt
I think we should do the following:
1) Mark IE support as a "partial_implementation": true and add a "notes": "Support for WebDriver is obtained via the InternetExplorerDriver tool [https://github.com/SeleniumHQ/selenium/wiki/InternetExplorerDriver]" (or similar).
2) Keep the false value for Edge but add a note: "The vendor-supported implementation for Edge is not spec-conforming and isn't compatible."

@Elchi3 what do you think of this?

Reply:
Sounds good to me. We could also reach out to the Microsoft folks to see what say think here.

Reply:
Support for IE isn’t partial, it is full. I don’t think we ought to discriminate against non-vendor implementations here.

Reply:
So we should update all of the IE support versions to 7. I don't think we need to point to all of the driver tools for the different browsers. Use of those is expected by anyone trying to use WebDriver.

I'd also be happy to add a note about non-conforming support for browsers, if @andreastt thinks it would be useful. I think it is useful to know there is support there, but it is non-standard, rather than just no support at all?

Reply:
Are there any other cases similar to Edge that we also need to report?

Reply:
Let me try to elaborate on the current situation. Certain browsers’ implementations are based on an earlier open source protocol called the Selenium JSON wire protocol. This protocol is being replaced by the W3C WebDriver protocol, and is so far implemented and shipped in Firefox and IE.

Through various shims, a wide range of clients offer forward- and backward compatibility with _both protocols_. A user might plausibly not observe any difference using WebDriver with a conforming remote end vs. a non-conforming remote end.

Whilst there is ongoing work with transitioning Chrome, Edge, and Safari to the new protocol, they do not ship with this new behaviour enabled by default. It is because of this that I think it is problematic for BCD to claim they are compatible when that compatibility is only a happenstance due to diligent client maintainers.

MDN documents the web platform, not arbitrary tools for interacting with it. I would be fine with adding a note saying that there exists an implementation but that it is non-conforming, but I would be anxious if that meant they turn up as green in the matrix chart.

I hope this helps.

Reply:
This does help - thanks! So to make it clear, specifying support added false with a note about the non-conforming implementation means it will turn up as red in the matrix. No worries there.

From what you've said above, it looks like I need to add the note about a non-conforming implementation being present to Chrome/Opera/Samsung, Edge, and Safari? Correct?

As well as updating IE support to 7, of course.

Reply:
Correct.